web/packages/teleport/src/Discover/Database/MutualTls/MutualTls.tsx

```diff
@@ -157,7 +157,10 @@ function DbEngineInstructions({ dbEngine }: { dbEngine: DatabaseEngine }) {
           ]}
         />
         <Text mt={1}>
-          Restart the database server to apply the configuration.
+          Restart the database server to apply the configuration. The
+          certificate is by default 90 days so this will require installing an
```

Comment:
```suggestion
          certificate is valid for 90 days so this will require installing an
```

I guess the next question is how can they generate a new one? Should we add the `tctl sign ...` command here? Or just link to docs?

Reply:
I think docs as they are things like is the user set up to impersonate right. I'll add that link

Reply:
@marcoandredinis , please see update